dev-workflow-ui/webContent/view/home.xhtml

```diff
@@ -80,7 +80,7 @@
               <p:dataTable id="activeTasks" var="task" value="#{homepageIvyDevWfBean.tasksDataModel}" lazy="true"
```

Comment:
I just noticed. 
I think this should be `personalTasksIvyDevWfBean.tasksDataModel`

Reply:
Yes I think you're right!